src/main/java/hudson/plugins/jira/JiraBuildAction.java

```diff
@@ -11,11 +15,12 @@
  * @author Kohsuke Kawaguchi
  */
 public class JiraBuildAction implements Action {
-    public final AbstractBuild<?, ?> owner;
+    
+	public final Run<?, ?> owner;
```

Comment:
Regarding the serialization, the right way in new Jenkins API is to implement RunAction2 interface andmake the field transient